vyper/builtins/functions.py

```diff
@@ -1061,6 +1061,25 @@ def fetch_call_return(self, node):
 
         kwargz = {i.arg: i.value for i in node.keywords}
 
+        delegate_call = kwargz.get("is_delegate_call")
+        static_call = kwargz.get("is_static_call")
+        if delegate_call and static_call:
+            raise ArgumentException(
+                "Call may use one of `is_delegate_call` or `is_static_call`, not both"
+            )
+
+        value = kwargz.get("value")
+        if (delegate_call or static_call) and value is not None:
+            raise ArgumentException("value= may not be passed for static or delegate calls!")
+
+        fn_node = node.get_ancestor(vy_ast.FunctionDef)
```

Comment:
note for refactoring: ideally we should pass this in a context object

Reply:
should I add a comment here?

Reply:
sure